### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by hamhands**
### What changed
- Added wage requirement faq entry

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,3 +14,5 @@
 <div id="disqus_thread">
 </div>
 
+
```
Comment (bsmithgall) [inline index.html]: ?

Comment (bsmithgall) [inline index.html]: fix whitespace problems like this forever: https://github.com/randy3k/whitespace

Comment (hamhands) [inline index.html]: https://github.com/codeforamerica/cleaning-pgh/commit/bfc473500de015b99233057f45610abe29256191

**Comment by hamhands** [comment]
Can I merge now?

**Pull request merged**